data/pokedex.js

```diff
@@ -10217,7 +10217,35 @@ exports.BattlePokedex = {
 		prevo: "spewpa",
 		evoLevel: 12,
 		eggGroups: ["Bug"],
-		otherForms: ["vivillonarchipelago", "vivilloncontinental", "vivillonelegant", "vivillongarden", "vivillonhighplains", "vivillonicysnow", "vivillonjungle", "vivillonmarine", "vivillonmodern", "vivillonmonsoon", "vivillonocean", "vivillonpolar", "vivillonriver", "vivillonsandstorm", "vivillonsavanna", "vivillonsun", "vivillontundra"],
+		otherForms: ["vivillonarchipelago", "vivilloncontinental", "vivillonelegant", "vivillonfancy", "vivillongarden", "vivillonhighplains", "vivillonicysnow", "vivillonjungle", "vivillonmarine", "vivillonmodern", "vivillonmonsoon", "vivillonocean", "vivillonpolar", "vivillonriver", "vivillonsandstorm", "vivillonsavanna", "vivillonsun", "vivillontundra", "vivillonpokeball"],
+	},
+	vivillonfancy: {
+		num: 666,
+		species: "Vivillon-Fancy",
+		baseSpecies: "Vivillon",
+		forme: "Fancy",
+		formeLetter: "F",
+		types: ["Bug", "Flying"],
+		baseStats: {hp: 80, atk: 52, def: 50, spa: 90, spd: 50, spe: 89},
+		abilities: {0: "Shield Dust", 1: "Compound Eyes"},
+		heightm: 1.2,
+		weightkg: 17,
+		color: "Black",
+		eggGroups: ["Undiscovered"],
```

Comment:
Well, that's true, but we probably shouldn't use `eggGroups` to mark that. We have `formats-data.js` for that.